test/runtests.jl

```diff
@@ -19,6 +20,7 @@ include("test_utils.jl")
     include("anchoredinterval.jl")
     include("comparisons.jl")
     include("plotting.jl")
+    include("arrow.jl")
```

Comment:
```suggestion
    VERSION >= v"1.3" && include("arrow.jl")
```

Reply:
Again, this isn't really helpful because we still need to declare Arrow.jl in our Project.toml.

Reply:
Okay, included a comment at the top of the file to explain why this is here. Since it probably isn't very intuitive why that would be desired.